tests/test_list_page.py

```diff
@@ -0,0 +1,443 @@
+import unittest
+import re
+from hamcrest import *
```

Comment:
What assertions does hamcrest provide that are not already provided in the built-in unittest package?

Reply:
I find the unittest.assertEqual() unusable due to its error messages. I use hamcrest to provide a message like "Expected 'Dec. 3rd'  but was  '12-03'"